src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -113,6 +122,63 @@ boot_spd_initaep_get(spdid_t spdid)
 	return cos_sched_aep_get(boot_spd_defcompinfo_get(spdid));
 }
 
+/* The fault handlers*/
+void
+fh_div_by_zero_err(unsigned long sp, unsigned long ip, unsigned long fault_addr, unsigned long fault_type)
+{
+	PRINTLOG(PRINT_DEBUG, "in div by zero error fault handler, fault happens in component:%u\n\n", cos_inv_token());
+	call_cap_op(0, 0, 0, 0, 0, 0);
```

Comment:
Just returning from a function should automatically call `sret`.  I'm not sure I understand the motivation for having the `call_cap_op`s here.